persistence-modules/spring-data-jpa-3/README.md

```diff
@@ -0,0 +1,6 @@
+=========
```

Comment:
You can omit this file for now. I will add it after the article goes live.

Reply:
Resolved.